closure/protobuf/closure_proto_library.bzl

```diff
@@ -89,7 +89,7 @@ closure_proto_aspect = aspect(
     attrs = dict({
         # internal only
         "_protoc": attr.label(
-            default = Label("@com_google_protobuf//:protoc"),
+            default = configuration_field(fragment = "proto", name = "proto_compiler"),
```

Comment:
Can we add a comment with more details around the benefit of this approach and what flag in `.bazelrc` would be used, etc.? It will be really useful for us or someone else in the future.

Reply:
Done